### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#218] [Q&A]
Hey @Caverns4 can you help me implement a cool bonus based off of sonic mania

**Comment by r6915ee** [comment]
...Are you entirely sure this is the best place to do such??

**Reply by Raptorq1** [reply]
> ...Are you entirely sure this is the best place to do such??

My bad

**Reply by Caverns4** [reply]
Look at how the time and Ring bonuses are implemented. The logic for them is extremely simple.
